docs/lava-blockchain/join-testnet-manual.md

```diff
@@ -215,6 +215,8 @@ Versions are tracked in [Lava git](https://github.com/lavanet/lava) (build from
 | v0.14.0 | 254645 |
 | v0.15.1 | 266490 |
```

Comment:
@David-Melamed @nirtayeb All these versions below v0.16.0 need to be deleted.
They do not belong here anymore, since we forked to Testnet-2.